### ISSUE TITLE
SVG fill-rule broken or not working correctly

### ISSUE THREAD

**Issue by Ericky14** [react-native-community/react-native-svg#1072]
# Bug

I have this SVG here which renders correctly on a browser (with 3 white bars), but does not render correctly on react-native. I assume it's because the `fill-rule` property is not working since when I remove that property on the browser, it behaves the same way as `react-native`.

Browser:
![image](https://user-images.githubusercontent.com/21248862/63107711-75241300-bf53-11e9-9b69-153ec18143e9.png)

React-native:
![image](https://user-images.githubusercontent.com/21248862/63107728-7bb28a80-bf53-11e9-914f-f452936e1984.png)

## Environment info

React native info output:
```bash
System:
    OS: macOS 10.14.5
    CPU: (12) x64 Intel(R) Core(TM) i7-8850H CPU @ 2.60GHz
    Memory: 29.31 MB / 16.00 GB
    Shell: 3.2.57 - /bin/bash
  Binaries:
    Node: 10.15.3 - ~/.nvm/versions/node/v10.15.3/bin/node
    Yarn: 1.16.0 - /usr/local/bin/yarn
    npm: 6.10.3 - ~/.nvm/versions/node/v10.15.3/bin/npm
    Watchman: 4.9.0 - /usr/local/bin/watchman
  SDKs:
    iOS SDK:
      Platforms: iOS 12.1, macOS 10.14, tvOS 12.1, watchOS 5.1
    Android SDK:
      Android NDK: 20.0.5594570
  IDEs:
    Android Studio: 3.1 AI-173.4907809
    Xcode: 10.1/10B61 - /usr/bin/xcodebuild
  npmPackages:
    react: 16.8.6 => 16.8.6 
    react-native: 0.60.3 => 0.60.3 
  npmGlobalPackages:
    react-native-cli: 2.0.1
    react-native-create-library: 3.1.2
    react-native-portal-library: 1.0.0
    react-native: 0.60.4
```

Library version: 9.6.2

## Steps To Reproduce

Use the svg component provided

Describe what you expected to happen:

Render svg correctly

## Reproducible sample code

Raw svg:
```
<svg width="32" height="32" viewBox="0 0 32 32" fill="none" xmlns="http://www.w3.org/2000/svg">
<path fill-rule="evenodd" clip-rule="evenodd" d="M4 0C1.79086 0 0 1.79086 0 4V28C0 30.2091 1.79086 32 4 32H28C30.2091 32 32 30.2091 32 28V4C32 1.79086 30.2091 0 28 0H4ZM17 6C17 5.44772 17.4477 5 18 5H20C20.5523 5 21 5.44772 21 6V25C21 25.5523 20.5523 26 20 26H18C17.4477 26 17 25.5523 17 25V6ZM12 11C11.4477 11 11 11.4477 11 12V25C11 25.5523 11.4477 26 12 26H14C14.5523 26 15 25.5523 15 25V12C15 11.4477 14.5523 11 14 11H12ZM6 18C5.44772 18 5 18.4477 5 19V25C5 25.5523 5.44772 26 6 26H8C8.55228 26 9 25.5523 9 25V19C9 18.4477 8.55228 18 8 18H6ZM24 14C23.4477 14 23 14.4477 23 15V25C23 25.5523 23.4477 26 24 26H26C26.5523 26 27 25.5523 27 25V15C27 14.4477 26.5523 14 26 14H24Z" fill="url(#paint0_linear)"/>
<defs>
<linearGradient id="paint0_linear" x1="0" y1="0" x2="8.46631" y2="37.3364" gradientUnits="userSpaceOnUse">
<stop stop-color="#FEA267"/>
<stop offset="1" stop-color="#E75A4C"/>
</linearGradient>
</defs>
</svg>
```

Component svg:
```
const renderSvg = () => (
  <Svg width={32} height={32} viewBox="0 0 32 32" fill="none">
    <Path fillRule="evenodd" clipRule="evenodd" fill="url(#paint0_linear)" d="M4 0C1.79086 0 0 1.79086 0 4V28C0 30.2091 1.79086 32 4 32H28C30.2091 32 32 30.2091 32 28V4C32 1.79086 30.2091 0 28 0H4ZM17 6C17 5.44772 17.4477 5 18 5H20C20.5523 5 21 5.44772 21 6V25C21 25.5523 20.5523 26 20 26H18C17.4477 26 17 25.5523 17 25V6ZM12 11C11.4477 11 11 11.4477 11 12V25C11 25.5523 11.4477 26 12 26H14C14.5523 26 15 25.5523 15 25V12C15 11.4477 14.5523 11 14 11H12ZM6 18C5.44772 18 5 18.4477 5 19V25C5 25.5523 5.44772 26 6 26H8C8.55228 26 9 25.5523 9 25V19C9 18.4477 8.55228 18 8 18H6ZM24 14C23.4477 14 23 14.4477 23 15V25C23 25.5523 23.4477 26 24 26H26C26.5523 26 27 25.5523 27 25V15C27 14.4477 26.5523 14 26 14H24Z"></Path>
    <Defs>
      <LinearGradient id="paint0_linear" x1="0" y1="0" x2="8.46631" y2="37.3364" gradientUnits="userSpaceOnUse">
        <Stop stopColor="#FEA267"></Stop>
        <Stop offset="1" stopColor="#E75A4C"></Stop>
      </LinearGradient>
    </Defs>
  </Svg>
);
```

**Comment by msand** [comment]
@Ericky14 Can you try with https://github.com/react-native-community/react-native-svg/commit/f2b0e257d1956a6d8fcf8077a80d3d1a4003d7c9

**Comment by msand** [comment]
```jsx
import * as React from 'react';
import { Svg, Path, Defs, LinearGradient, Stop } from 'react-native-svg';

export default () => (
  <Svg width="100%" height="100%" viewBox="0 0 32 32">
    <Path
      fillRule="evenodd"
      clipRule="evenodd"
      fill="url(#paint0_linear)"
      d="M4 0C1.79086 0 0 1.79086 0 4V28C0 30.2091 1.79086 32 4 32H28C30.2091 32 32 30.2091 32 28V4C32 1.79086 30.2091 0 28 0H4ZM17 6C17 5.44772 17.4477 5 18 5H20C20.5523 5 21 5.44772 21 6V25C21 25.5523 20.5523 26 20 26H18C17.4477 26 17 25.5523 17 25V6ZM12 11C11.4477 11 11 11.4477 11 12V25C11 25.5523 11.4477 26 12 26H14C14.5523 26 15 25.5523 15 25V12C15 11.4477 14.5523 11 14 11H12ZM6 18C5.44772 18 5 18.4477 5 19V25C5 25.5523 5.44772 26 6 26H8C8.55228 26 9 25.5523 9 25V19C9 18.4477 8.55228 18 8 18H6ZM24 14C23.4477 14 23 14.4477 23 15V25C23 25.5523 23.4477 26 24 26H26C26.5523 26 27 25.5523 27 25V15C27 14.4477 26.5523 14 26 14H24Z"
    />
    <Defs>
      <LinearGradient
        id="paint0_linear"
        x1="0"
        y1="0"
        x2="8.46631"
        y2="37.3364"
        gradientUnits="userSpaceOnUse">
        <Stop stopColor="#FEA267" />
        <Stop offset="1" stopColor="#E75A4C" />
      </LinearGradient>
    </Defs>
  </Svg>
);

```

**Comment by Ericky14** [comment]
That works for me, thank you. When do you think the new version will be released?

**Comment by msand** [comment]
@Ericky14 published v9.6.3 now
